packages/manager/modules/manager-pci-common/src/components/input-adapter/radio-adapter/RadioField.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC, PropsWithChildren, ReactNode } from 'react';
+import { Subtitle } from '@ovh-ux/manager-react-components';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+
+export type RadioFieldProps = PropsWithChildren<{
+  label: ReactNode;
+  subtitle?: ReactNode;
+}>;
+
+export const RadioField: FC<RadioFieldProps> = ({
```

Comment:
it doesn't look specific to pci to me, shouldn't this be in manager react components ?

Reply:
Yes, most of it should be in it but now we can't add more features to MRC in ODS17, only fix, as far as I know

Reply:
so you are pushing new features to the maintenance branch of pci-common, shouldn't the branch be used for maintenance only ? do you plan to backport thoses changes to the develop branch as well ?